README.md

```diff
@@ -0,0 +1,35 @@
+# Pearl Pressure Controller Support and Emulator 
+
+The PEARL pressure controller is an ISIS-designed device. 
+
+## Pearl Pressure Controller Useful Recourses
+
+### Technical Manual Location: 
+
+`\\isis\shares\ISIS_Experiment_Controls\Manuals\PEARL Pressure Controller\PEARL handbook_v3_5.docx`
+
+### LabVIEW VI Location: 
+
+`C:\LabVIEW Modules\Instruments\PEARL\PEARL Pressure Cell Controller`
+
+### WIKI Pages: 
+
+https://github.com/ISISComputingGroup/IBEX/wiki/PEARL-Instrument-Details
+
+### IOC Repository: 
+
+https://github.com/ISISComputingGroup/EPICS-ioc/tree/4cd6b6a481666ea77c155126b5114b4ca3714eb3/PEARLPC
```

Comment:
Why this commit and not master?

Reply:
Updated to master branch. This is something I missed as I just copied the URL I was take too when navigating from https://github.com/ISISComputingGroup/EPICS/tree/master/ioc into IOC sub-module.